update-docs.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs-extra')
+const markdownTable = require('markdown-table')
+
+/**
+ * This script will update a section of the README.md file with an auto-
+ * generated markdown table listing all of the top level keys that can be
+ * defined in the manifest.json file.
+ */
+async function main () {
+  const jsonSchema = require('./manifest-json-schema.json')
+  const tableData = [
+    [ 'Key', 'Type', 'Required', 'Default', 'Description' ]
+  ]
+
+  Object.keys(jsonSchema.properties).forEach(key => {
+    const keyData = jsonSchema.properties[key]
+    tableData.push([
+      key,
+      keyData.type,
+      (jsonSchema.required || []).includes(key) ? 'Required' : 'Optional',
+      keyData.default,
+      keyData.description
+    ])
+  })
+
+  let readmeContent = await fs.readFile('./README.md', { encoding: 'UTF-8' })
+  readmeContent = readmeContent.substring(0, readmeContent.indexOf('### Manifest.json values') + 26)
```

Comment:
Is there a way to change this `indexOf` search to a hidden id or something so that if the header name changes, we won't be in a situation where this script breaks?

Reply:
Also, I think this probably needs a travis script that runs update-docs and compares to README to verify that docs are updated.

Reply:
I originally attempted to use a markdown parsing library to be more smart about all this, but it was taking too long to figure out, so I opted for this simpler tactic instead.